### ISSUE TITLE
Fix getenv warning when compiling for Microsoft Windows.

### ISSUE THREAD

**Issue by IAmNotHanni** [inexorgame/entity-system#118]
Everytime I compile I get the following warning:

```
warning C4996: 'getenv': This function or variable may be unsafe.
Consider using _dupenv_s instead.
To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
```

**Comment by IAmNotHanni** [comment]
https://stackoverflow.com/questions/48568707/getenv-this-function-or-variable-may-be-unsafe-really

**Comment by IAmNotHanni** [comment]
https://stackoverflow.com/questions/15916695/can-anyone-give-me-example-code-of-dupenv-s

**Comment by IAmNotHanni** [comment]
We could use `std::getenv`
https://de.cppreference.com/w/cpp/utility/program/getenv

**Comment by aschaeffer** [comment]
Fixed in e2d627fa3dedbf6ced1a3d3bb4fa3932bf93d4e5
